src/utils/data.js

```diff
@@ -39,14 +40,31 @@ export async function getSheetsData(sheetId) {
   return d3.csvParse(text, d3.autoType);
 }
 
-export async function getData() {
+export async function getData(config) {
+  // TODO: Get the translation language based on the translator class instance
+  // TODO: Get translations based on this language, currently only getting from query params
+  const translations = await Translator.getTranslation("i18n", config.lang);
```

Comment:
There's no error handling of `getTranslation` here, so if it errors I think it may prevent the rest of the code below from running. 

It also seems odd to do it here in `getData`, I think because `getData()` is referring to the map's data, not the translation files. I think what might make more sense is to have the initial `getTranslation` called in the constructor of the `Translator` class. That way, when it is instantiated, it will automatically grab the translation file, if necessary. Grabbing it first from here seems a bit out of place.